packages/core/src/env-set/index.ts

```diff
@@ -12,14 +15,35 @@ import createQueryClient from './create-query-client.js';
 import loadOidcValues from './oidc.js';
 import { throwNotLoadedError } from './throw-errors.js';
 
-export enum MountedApps {
+/** Apps (also paths) for user tenants. */
+export enum UserApps {
   Api = 'api',
   Oidc = 'oidc',
   Console = 'console',
   DemoApp = 'demo-app',
   Welcome = 'welcome',
 }
 
+/** Apps (also paths) ONLY for the admin tenant. */
+export enum AdminApps {
+  Me = 'me',
```

Comment:
I think `console` and `welcome` should be in the `AdminApps` enum

Reply:
yes